genesis-volatility/src/adapter.ts

```diff
@@ -0,0 +1,52 @@
+import { ExecuteFactory, ExecuteWithConfig } from '@chainlink/types'
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { makeConfig } from './config'
+
+const customParams = {
+  symbol: ['base', 'from', 'coin', 'symbol'],
+  days: ['days', 'period'],
+}
+
+export const execute: ExecuteWithConfig = async (input, config) => {
```

Comment:
No need to export this fn anymore now that we export `makeExecute`

```suggestion
const execute: ExecuteWithConfig = async (input, config) => {
```